plugins/hls-tactics-plugin/src/Wingman/LanguageServer.hs

```diff
@@ -77,18 +83,27 @@ runStaleIde state nfp a = MaybeT $ runIde state $ useWithStale a nfp
 
 
 ------------------------------------------------------------------------------
--- | Get the the plugin config
-getTacticConfig :: MonadLsp Plugin.Config m => ShakeExtras -> m Config
-getTacticConfig extras = do
-  pcfg <- getPluginConfig extras "tactics"
-  pure $ case fromJSON $ Object $ plcConfig pcfg of
-    Success cfg -> cfg
-    Error _     -> emptyConfig
 
+properties :: Properties
+  '[PropertyKey "max_use_ctor_actions" 'TNumber,
+    PropertyKey "features" 'TString]
+properties = emptyProperties
+  & defineStringProperty @"features"
+    "Features set used by wingman (tactic) plugin" ""
```

Comment:
```suggestion
    "Feature set used by Wingman" ""
```

Reply:
Is there a heading in the configuration for each plugin?

Reply:
Sorry, what does "heading" mean here?

Reply:
Is it possible to organize our config in VS code like the following:

# Haskell

## General

* Foo
* Bar
* Baz

## Brittany Plugin


* Foo
* Bar
* Baz

## Wingman Plugin


* Foo
* Bar
* Baz

Reply:
Oh I see, thanks for explanation. But unfortunately it seems impossible :/

> The title is the main heading that will be used for your configuration section. Normally you will only have one section for your extension.